provider/aci.go

```diff
@@ -657,6 +661,19 @@ func (p *ACIProvider) CreatePod(ctx context.Context, pod *v1.Pod) error {
 	containerGroup.Location = p.region
 	containerGroup.RestartPolicy = aci.ContainerGroupRestartPolicy(pod.Spec.RestartPolicy)
 	containerGroup.ContainerGroupProperties.OsType = aci.OperatingSystemTypes(p.operatingSystem)
+	if pod.Annotations != nil {
+		priority, priorityExists := pod.Annotations[priorityTypeAnnotation]
+		if priorityExists {
+
+			if strings.EqualFold(priority, string(aci.Spot)) {
+				containerGroup.ContainerGroupProperties.Priority = aci.Spot
+			} else if strings.EqualFold(priority, string(aci.Regular)) {
+				containerGroup.ContainerGroupProperties.Priority = aci.Regular
+			} else {
```

Comment:
won't this fail all the existing ones?

Reply:
Validated that this change does not affect existing behavior